File: #Folder Pertemuan 1/Konversi_Bilangan/Form1.cs
using Microsoft.VisualBasic;
using System.Windows.Forms;
namespace Konversi_Suhu
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void lblCelcius_Click(object sender, EventArgs e)
        {

        }

        private void txtInput_TextChanged(object sender, EventArgs e)
        {

        }

        private void btnKonversi_Click(object sender, EventArgs e)
        {
            double celcius, fahrenheit, reamur, kelvin;
            celcius = Convert.ToInt64(txtInput.Text);
            fahrenheit = 9 / 5 * celcius + 32;
            reamur = 4 / 5 * celcius;
            kelvin = celcius + 273;

            
        }

        private void btnKeluar_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }
    }
}

File: #Folder Pertemuan 1/Konversi_Bilangan/Form1.vb
Public Class Form1
    Private Sub lblCelcius_Click(sender As Object, e As EventArgs) Handles lblCelcius.Click

    End Sub

    Private Sub txtInput_TextChanged(sender As Object, e As EventArgs) Handles txtInput.TextChanged

    End Sub

    Private Sub btnKonversi_Click(sender As Object, e As EventArgs) Handles btnKonversi.Click
        Dim celcius, fahrenheit, reamur, kelvin As Double
        celcius = Val(txtInput.Text)
        fahrenheit = 9 / 5 * celcius + 32
        reamur = 4 / 5 * celcius
        kelvin = celcius + 273
        MsgBox("Fahrenheit = " & fahrenheit & vbCrLf & "Reamur = " & reamur & vbCrLf & "Kelvin = " & kelvin)

    End Sub

    Private Sub btnKeluar_Click(sender As Object, e As EventArgs) Handles btnKeluar.Click
        Me.Close()
    End Sub
End Class
